src/internal/packager/helm/chart.go

```diff
@@ -63,49 +62,18 @@ func (h *Helm) InstallOrUpgradeChart() (types.ConnectStrings, string, error) {
 		return nil, "", fmt.Errorf("unable to create helm renderer: %w", err)
 	}
 
-	attempt := 0
-	for {
-		attempt++
-
-		histClient := action.NewHistory(h.actionConfig)
+	// TODO: DO_NOT_SUBMIT
```

Comment:
@Racer159 thoughts here? It looks safe to reuse but the history client does have a storage thing.

Reply:
It should be good to reuse - the action config is what has the releases storage thing and we are already reusing that as `h.actionConfig`